install/overlays/vsphere/kustomization.yaml

```diff
@@ -36,8 +59,12 @@ secretGenerator:
 - name: peer-pods-secret
   namespace: confidential-containers-system
   literals:
-  - GOVC_USERNAME="" # set
-  - GOVC_PASSWORD="" # set
+  - GOVC_USERNAME=""   # set of the vCenter username is required.
+
+  - GOVC_PASSWORD=""   # set of the vCenter password is required.
+
```

Comment:
nit: `set of the vCenter` -> `set if the vCenter`

Reply:
The set is required. There is no if.  I could say # The setting of the vCenter blah is required.

Reply:
Or  'Setting the vCenter blah is required'